app/src/main/java/com/mapbox/mapboxsdk/plugins/testapp/activity/offline/OfflineDownloadActivity.java

```diff
@@ -32,174 +19,34 @@
  */
 public class OfflineDownloadActivity extends AppCompatActivity {
 
-  @BindView(R.id.edittext_region_name)
-  EditText regionNameView;
-
-  @BindView(R.id.edittext_lat_north)
-  EditText latNorthView;
-
-  @BindView(R.id.edittext_lon_east)
-  EditText lonEastView;
-
-  @BindView(R.id.edittext_lat_south)
-  EditText latSouthView;
-
-  @BindView(R.id.edittext_lon_west)
-  EditText lonWestView;
-
-  @BindView(R.id.min_text_view)
-  TextView minTextView;
-
-  @BindView(R.id.max_text_view)
-  TextView maxTextView;
-
-  @BindView(R.id.spinner_style_url)
-  Spinner styleUrlView;
-
-  @BindView(R.id.seekbar_min_zoom)
-  SeekBar minZoomView;
-
-  @BindView(R.id.seekbar_max_zoom)
-  SeekBar maxZoomView;
+  private static final int REQUEST_CODE = 9384;
 
```

Comment:
Can we still keep the form around? This is useful for reproducing client bugs with specific bounds/style. The UI selector could be a separate activity in the plugin overview.